install_kafka.sh

```diff
@@ -1,26 +1 @@
-cp server.properties server.properties1
-sed -i.bak 's/@broker_id@/1/g' server.properties1
-sed -i.bak 's/@port@/9092/g' server.properties1
-cp server.properties server.properties2
-sed -i.bak 's/@broker_id@/2/g' server.properties2
-sed -i.bak 's/@port@/9093/g' server.properties2
-cp server.properties server.properties3
-sed -i.bak 's/@broker_id@/3/g' server.properties3
-sed -i.bak 's/@port@/9094/g' server.properties3
-mkdir kafka
-rm -rf /tmp/zookeeper
-rm -rf /tmp/kafka-logs*
```

Comment:
Travis starts a fresh container for every run, which means a fresh /tmp. So the rm actions are a no-op.